code/game/machinery/cloning.dm

```diff
@@ -203,7 +203,7 @@
 
 		H.set_cloned_appearance()
 
-		H.suiciding = FALSE
+		H.remove_trait(TRAIT_NOREVIVE, SUICIDING)

```

Comment:
this is causing 36205 why does this exist...?

Reply:
i'm not even sure this is ever going to do anything considering it's a new body. I kinda just converted everything regardless of how much sense it made to avoid weird issues.

Reply:
the cloning process previously checked for suiciding when processing. If this was removed, it would eject the individual out (as it should).

When I asked " why does this exist " I was speaking from a general perspective. I understand you just converted it into your traits :^)